stubs/Markdown/markdown/core.pyi

```diff
@@ -30,7 +31,7 @@ class Markdown:
     def registerExtensions(self, extensions: Sequence[Extension | str], configs: Mapping[str, Mapping[str, Any]]) -> Markdown: ...
     def build_extension(self, ext_name: Text, configs: Mapping[str, str]) -> Extension: ...
     def registerExtension(self, extension: Extension) -> Markdown: ...
-    def reset(self: Markdown) -> Markdown: ...
+    def reset(self: Self) -> Self: ...
```

Comment:
This is semantically different to the stub as it is currently, but looks more correct to me. https://github.com/Python-Markdown/markdown/blob/1d41f13c774696d651921601c827ed500e2aa285/markdown/core.py#L190